BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -369,6 +369,7 @@ public BrainCloudComms(BrainCloudClient client)
         /// <param name="secretKey">Secret key.</param>
         public void Initialize(string serverURL, string appId, string secretKey)
         {
+            ResetCommunication();
             _packetId = 0;
```

Comment:
If _packetId is reset, you have to reset _sessionId also. They go hand in hand.

Reply:
SessionId is reset in the resetCommunication

Reply:
Then move _packetId = 0 inside ResetCommunication.